packages/components/src/Page/Page.tsx

```diff
@@ -32,21 +34,51 @@ interface PageProps {
    * @default standard
    */
   readonly width?: "fill" | "standard" | "narrow";
+
+  /**
+   * Page title primary action button settings.
+   */
+  readonly primaryAction?: ButtonProps;
+
+  /**
+   * Page title secondary action button settings.
+   */
+  readonly secondaryAction?: ButtonProps;
```

Comment:
@chris-at-jobber Do you think we should provide a default button `type` for secondary?

Reply:
I don't think I understand what you're asking, `secondary` is a` type` as far as I am aware

Reply:
If the question is "should the secondary button have a `type` of `secondary`?" then yes 😄